src/main/java/com/ksm/wstbackoffice/controller/handler/ControllerExceptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.ksm.wstbackoffice.controller.handler;
+
+import com.ksm.wstbackoffice.exception.ErrorResponseMessage;
+import com.ksm.wstbackoffice.exception.ResourceNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.time.LocalDate;
+
+@ControllerAdvice
+public class ControllerExceptionHandler {
+    @ExceptionHandler(ResourceNotFoundException.class)
+    public ResponseEntity<ErrorResponseMessage> resourceNotFoundException(ResourceNotFoundException ex) {
+        ErrorResponseMessage message = new ErrorResponseMessage(
+                LocalDate.now(),
```

Comment:
Why do you think a user wants to have this information?

Reply:
useful for log info in feature